tests/test_auth_utils.py

```diff
@@ -0,0 +1,253 @@
+import datetime
+
+import fastapi
+import jwt
+import pytest
+from Crypto.PublicKey import RSA
+
+from huma_utils import auth_utils, constants, datetime_utils
+from huma_utils.test_helpers import address_helpers
+
+
+def describe_verify_wallet_ownership() -> None:
+    @pytest.fixture(scope="session")
+    def rsa_key() -> RSA.RsaKey:
+        return RSA.generate(2048)
+
+    @pytest.fixture
+    def wallet_address() -> str:
+        return address_helpers.fake_hex_address()
+
+    @pytest.fixture
+    def chain_id() -> str:
+        return "1"
+
+    @pytest.fixture
+    def claim_sub(wallet_address: str, chain_id: str) -> str:
+        return f"{wallet_address}:{chain_id}"
+
+    @pytest.fixture
+    def token_expiration_time() -> datetime.datetime:
+        return datetime_utils.tz_aware_utc_now() + datetime.timedelta(days=3)
```

Comment:
Question: so the JWT token works for 3 days. After this period, users need to resign the wallet?

Reply:
That's right